data/wanderer/wanderer outfits.txt

```diff
@@ -301,8 +301,8 @@ outfit "Small Biochemical Cell"
 	thumbnail "outfit/small biochemical"
 	"mass" 17
 	"outfit space" -17
-	"energy generation" 1.6
-	"heat generation" 1.4
+	"energy generation" 1.28
+	"heat generation" 1.12
 	"energy capacity" 7500
```

Comment:
I think that the battery capacity is the thing thats too high here not the energy generation
removing a bit of energy gen really doesnt matter, except it might bother some ships that run on these only?
6k battery feels like a reasonable amount to me (and 25k for the other) where it's still strong but I'm not sure if its op or not anymore

Reply:
7500 energy capacity in a 17 ton outfit isn't unreasonable. Four ka'het reserve accumulators would give you 9600 in 12 tons, the Crystal Capactitor provides 13600 in the equivalent of 14 space. As batteries, these aren't unreasonable, or even particularly good.
It's the combination of battery and generation taken together that makes the biochemical cells too strong, and I'd rather nerf the generation rather than the battery since they're already clearly more "batteries with a bit of generation" rather than "generators with a bit of battery", and I'd prefer to lean into that rather than watering it down.

Reply:
My worry isnt really balance wise, its lore wise and its to do with the interactions between uhai and wanderers, I ideally want the wand using these to counter ion to come at a cost (say require the big one at the very least) and not work too well against the ionic turrets either

Reply:
Batteries don't really 'counter' ion damage, they just allow a ship to weather it temporarily. Countering ion damage requires having enough generation to offset it. Sustained ion damage _will_ eventually wear down a ship that's relying on batteries to resist ion.

Reply:
Scrambling will work less effectively the more battery you have and you only need to weather the storm until you beat the enemy ship (with raw dmg too when you think about it)